app/lib/hooks/useVideoConf/StartACallActionSheet.tsx

```diff
@@ -0,0 +1,91 @@
+import { Camera, CameraType } from 'expo-camera';
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ESounds, useVideoConfRinger } from '.';
+import { useAppSelector } from '..';
+import { cancelCall, initVideoCall } from '../../../actions/videoConf';
+import AvatarContainer from '../../../containers/Avatar';
+import Button from '../../../containers/Button';
+import { CallHeader } from '../../../containers/CallHeader';
+import i18n from '../../../i18n';
+import { getUserSelector } from '../../../selectors/login';
+import { useTheme } from '../../../theme';
+import useUserData from '../useUserData';
+
+export default function StartACallActionSheet({ rid }: { rid: string }): React.ReactElement {
+	const { playSound, stopSound } = useVideoConfRinger(ESounds.DIALTONE, false);
+
+	const { colors } = useTheme();
+	const [mic, setMic] = useState(true);
+	const [cam, setCam] = useState(false);
+	const [containerWidth, setContainerWidth] = useState(0);
+
+	const username = useAppSelector(state => getUserSelector(state).username);
+	const calling = useAppSelector(state => state.videoConf.calling);
+	const dispatch = useDispatch();
+
+	const user = useUserData(rid);
+
+	return (
+		<View style={style.actionSheetContainer} onLayout={e => setContainerWidth(e.nativeEvent.layout.width / 2)}>
+			<CallHeader
+				title={calling ? i18n.t('Calling') : i18n.t('Start_a_call')}
+				cam={cam}
+				mic={mic}
+				setCam={setCam}
+				setMic={setMic}
+				avatar={user.avatar}
+				name={user.username}
+				uid={user.uid}
+				direct={user.direct}
+			/>
+			<View
+				style={[
+					style.actionSheetPhotoContainer,
+					{ backgroundColor: cam ? undefined : colors.conferenceCallPhotoBackground, width: containerWidth }
+				]}
+			>
+				{cam ? (
+					<Camera style={[style.cameraContainer, { width: containerWidth }]} type={CameraType.front} />
+				) : (
+					<AvatarContainer size={62} text={username} rid={rid} type={user.type} />
+				)}
+			</View>
+			<Button
```

Comment:
It should consider the safe area
<img src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/804994/715fa5cc-ac4b-417b-baa5-5f902f3dc3dc" width="350" />